pkg/scheduler/factory/plugins_test.go

```diff
@@ -51,22 +51,12 @@ func TestAlgorithmNameValidation(t *testing.T) {
 	}
 }
 
-func TestValidatePriorityConfigOverFlow(t *testing.T) {
+func TestValidatePriorityConfig(t *testing.T) {
 	tests := []struct {
 		description string
 		configs     []priorities.PriorityConfig
 		expected    bool
 	}{
-		{
```

Comment:
The following tests doesn't seem relevant anymore for me, as the weight size is `int32` and the total Node Priority score is `int`. So, they are only relevant in case of 32-bit architectures.  But when it comes to 64-bit architectures it is really hard to overflow having a weight of size `32 bit` and a total priority score of size `64 bit`. Any thoughts on this? Is it fine to just remove those cases? Is there some other test cases that would be relevant and could be added (I couldn't come up with any)?

Reply:
considering that we move to int64, the tests are still valid. I think the second one is valid regardless.

Reply:
There should still be a test for going over the limits.

Reply:
`PriorityConfig.Weight` (of type int32)*`schedulerapi.MaxPriority` (of value 10) wouldn't overflow in `int` type unless we have a very large number of PriorityConfigs in the test.

Reply:
if you set MaxTotalPriority to math.MaxInt32, then those tests should still be valid, right?

Reply:
ditto, moving everything to int64 should address this as well .